waspc/data/Generator/templates/db/schema.prisma

```diff
@@ -13,7 +13,7 @@ generator client {
 
 {=# entities =}
 model {= name =} {
-    {= pslModelSchema =}
+    {=& pslModelSchema =}
```

Comment:
This correctly HTML-escapes the schema, so if it has double quotes in it, those will work fine.